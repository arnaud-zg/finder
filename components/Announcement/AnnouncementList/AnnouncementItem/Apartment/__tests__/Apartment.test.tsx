import React from 'react'
import renderer from 'react-test-renderer'
import * as ApartmentIndex from '../'
import { Apartment } from '../'

describe('Apartment', () => {
  it('should take a snapshot of ApartmentIndex', () => {
    expect(ApartmentIndex).toMatchSnapshot()
  })

  it('renders correctly', () => {
    const testRenderer = renderer.create(
      <Apartment
        description="Ground floor Studio apartment with Balcony !"
        identifier="1"
        locationIndication="Paris"
        price={75000}
        publishTime="2019-09-29T13:50:42.926Z"
      />
    )
    expect(testRenderer.toJSON()).toMatchSnapshot()
  })
})
