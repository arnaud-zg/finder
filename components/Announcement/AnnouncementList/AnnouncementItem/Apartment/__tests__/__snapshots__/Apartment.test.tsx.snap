// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Apartment renders correctly 1`] = `
<div
  id="1"
>
  <h3
    className="fiird0z"
  >
    Ground floor Studio apartment with Balcony !
  </h3>
  <div
    className="f4lw1zx"
  >
    <p
      className="f2qtt8h"
    >
      Paris
    </p>
  </div>
  <div
    className="f18ec881"
  >
    750
     â‚¬
  </div>
  <div
    className="f18ec881"
  >
    29/09/2019 at 13:50
  </div>
</div>
`;

exports[`Apartment should take a snapshot of ApartmentIndex 1`] = `
Object {
  "Apartment": [Function],
}
`;
